ovs/extensions/healthcheck/helpers/helper.py

```diff
@@ -77,26 +77,22 @@ def get_local_settings():
                           'hostname': socket.gethostname(),
                           'storagerouter_id': Helper.LOCAL_ID,
                           'storagerouter_type': Helper.LOCAL_SR.node_type,
-                          'environment_release': ovs_version[0],
-                          'environment_branch': ovs_version[1].title(),
+                          'environment_release': ovs_version,
                           'environment os': ' '.join(platform.linux_distribution())}
         return local_settings
 
     @staticmethod
     def get_ovs_version():
```

Comment:
if we return the releasename why not rename this function too -> refactor -> rename